<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalChecker</class>
 <widget class="QWidget" name="GlobalChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Branchements</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arduino mère connectée</string>
    </property>
   </widget>
   <widget class="Led" name="arduino2Led">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arduino #3 connectée</string>
    </property>
   </widget>
   <widget class="Led" name="arduino1Led">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arduino #2 connectée</string>
    </property>
   </widget>
   <widget class="Led" name="webcamPluggedLed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Led" name="arduino3Led">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Webcam de surveillance
 branché</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la puissance</string>
    </property>
   </widget>
   <widget class="Led" name="powerLed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Led" name="commandLed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat des commande</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arduino #4 connectée</string>
    </property>
   </widget>
   <widget class="Led" name="arduino4Led">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Etats</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Boitier ouvert</string>
    </property>
   </widget>
   <widget class="Led" name="boxOpenLed">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>70</y>
      <width>57</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ATU</string>
    </property>
   </widget>
   <widget class="Led" name="tempICLed">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>110</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature 
carte électronique</string>
    </property>
   </widget>
   <widget class="Led" name="ATULed">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>70</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>--°C</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Polybox prête à l'emploi : &lt;/b&gt;</string>
   </property>
  </widget>
  <widget class="Led" name="polyboxReadyLed">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/img/img/led_off_red.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QLabel</extends>
   <header>Led.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
